Resources/assets/css/core.scss

```diff
@@ -190,3 +190,13 @@ aside {
         transition: background-color .2s cubic-bezier(.455, .03, .515, .955), color .2s cubic-bezier(.455, .03, .515, .955);
     }
 }
+
+/* Disabled style for div with form-control */
+div.form-control.disabled {
+    background-color: #e9ecef;
```

Comment:
It will be better to use `$input-disabled-bg;` here (defined in `bootstrap/scss/_forms.scss`).

Reply:
Yeah that would be better. The variable is now used.